mainpage/wlansi/map/templates/map/map.html

```diff
@@ -4,11 +4,6 @@
 {% addtoblock "js" %}<script type="text/javascript" src="{% static "wlansi/js/map.js" %}"></script>{% endaddtoblock %}
 {% addtoblock "js" %}<script type="text/javascript">
     /* <![CDATA[ */
-    var nodes = [
-        {% for node in nodes %}
-            {'lat': {{ node.geo_lat|unlocalize }}, 'long': {{ node.geo_long|unlocalize }}, 'name': '{{ node.name|escapejs }}', 'url': '{{ node.get_full_url|escapejs }}'}{% if not forloop.last %},{% endif %}
-        {% endfor %}
-    ];
     var MAP_LATITUDE = '{{ MAP_LATITUDE|escapejs }}';
     var MAP_LONGITUDE = '{{ MAP_LONGITUDE|escapejs }}';
     var MAP_ZOOM = {{ MAP_ZOOM }};
```

Comment:
You could create URLs to icons here as well: https://stackoverflow.com/questions/27932983/django-static-path-in-javascript-file